cloudformation/extraction.yaml

```diff
@@ -51,6 +55,35 @@ ExtractQueueDeadLetterQueue:
   Properties:
     KmsMasterKeyId: !GetAtt KmsKey.Arn
 
+TextractRole:
+  Type: AWS::IAM::Role
+  Properties:
+    AssumeRolePolicyDocument:
+      Version: "2012-10-17"
+      Statement:
+        - Effect: "Allow"
+          Principal:
+            Service:
+              - "textract.amazonaws.com"
+              - "lambda.amazonaws.com"
+          Action:
+            - "sts:AssumeRole"
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AWSLambdaExecute"
+    Policies:
+      - PolicyName: "TextractRoleAccess"
+        PolicyDocument:
+          Version: "2012-10-17"
+          Statement:
+            - Effect: Allow
+              Action:
+                - "sns:*"
+              Resource: "*"
```

Comment:
This should be locked down to the actual topic as well. Actions can very likely be limited as well.